pkg/storage/bloom/v1/versioned_builder.go

```diff
@@ -21,14 +21,14 @@ Builders provide the following methods:
 */
 
 // Convenience constructor targeting the most current version.
-func NewBlockBuilder(opts BlockOptions, writer BlockWriter) (*V2Builder, error) {
-	return NewBlockBuilderV2(opts, writer)
+func NewBlockBuilder(opts BlockOptions, writer BlockWriter) (*BlockBuilder, error) {
+	return NewBlockBuilderV3(opts, writer)
 }
 
 // Convenience alias for the most current version.
-type BlockBuilder = V2Builder
+type BlockBuilder = V3Builder
 
-type V2Builder struct {
```

Comment:
Will we support building older versions of blocks? If not, does it make sense to keep V3Builder in case we add a V4Builder in the future? Or should we name this Builder right away and keep changing it as we add versions?

Reply:
At some point when the feature is GA, yes. For now we can make all changes to the V3Builder.